mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -557,8 +598,33 @@
         <Arc value="0"/>
         <Range index="0" value="5"/>
     </CAbilEffectTarget>
+    <CAbilMorph id="LiberatorMorphtoAA">
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+            <SectionArray index="Mover">
+                <DurationArray index="Duration" value="0"/>
+            </SectionArray>
+            <SectionArray index="Stats">
+                <DurationArray index="Delay" value="0"/>
+            </SectionArray>
+        </InfoArray>
+    </CAbilMorph>
+    <CAbilMorph id="LiberatorMorphtoAG">
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
```

Comment:
**undocumented**: Fixed an issue where Liberators could not be given certain orders immediately after being ordered to siege
- Sieging Liberators had the ability transformation delay reduced from 0.5 to 0.0

This should allow immediately queueing of an unsiege command and might be necessary for queuing attack commands on ground units